test/sql/hnsw_dist_func.sql

```diff
@@ -0,0 +1,133 @@
+CREATE TABLE small_world_l2 (
+    id varchar(3),
+    vector real[]
+);
+
+CREATE TABLE small_world_cos (
+    id varchar(3),
+    vector real[]
+);
+
+CREATE TABLE small_world_ham (
+    id varchar(3),
+    vector integer[]
```

Comment:
What happens if I set this to `real[]` ?
does hamming operator refuse to work? does it return the wrong result?
If it is handled, let's add another column to this table (`vector_float`) and also create an index on that in the regression test.

Reply:
no, it will not work now, we should add operator class for `real[]` type for hamming distance in order to work.